src/main/java/org/assertj/core/util/DateUtil.java

```diff
@@ -22,6 +22,7 @@
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.time.LocalDate;
```

Comment:
I don't see it being used, if that's true remove this import

Reply:
I think there is no need to change this file for this PR, let's revert all the changes as they are not relevant to the PR